packages/web/components/Loading/__tests__/index.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { render } from 'test-utils';
+import Loading from '..';
+
+describe('Loading component', () => {
+	const childrenText = 'hi';
+
+	it('render loading or children correctly', () => {
+		const { container, rerender } = render(
+			<Loading loading={false}>
+				<h1>{childrenText}</h1>
+			</Loading>,
+		);
+
+		expect(container.querySelector('h1').textContent).toEqual(childrenText);
```

Comment:
É mais interessante usar as queries to react-testing-library, melhora as asserções e evita ter que ficar fazendo `querySelector` e usando `textContent` 

E.g `expect(screen.getByRole('heading')).toHaveTextContent(childrenText)`

Dessa forma quando o teste falhar o jest vai informar que o `textContent` não está sendo o valor esperado, no lugar de uma mensagem genérica.